docs/data/data-grid/column-groups/BasicGroupingDemo.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { DataGridPro, GridColDef } from '@mui/x-data-grid-pro';
+import { GridColumnGroupingModel } from '@mui/x-data-grid/models/gridColumnGrouping';
+
+const columns: GridColDef[] = [
+  { field: 'id', headerName: 'ID', width: 90 },
+  {
+    field: 'firstName',
+    headerName: 'First name',
+    width: 150,
+    editable: true,
+  },
+  {
+    field: 'lastName',
+    headerName: 'Last name',
+    width: 150,
+    editable: true,
+  },
+  {
+    field: 'age',
+    headerName: 'Age',
+    type: 'number',
+    width: 110,
+    editable: true,
+  },
+];
+
+const rows = [
+  { id: 1, lastName: 'Snow', firstName: 'Jon', age: 35 },
+  { id: 2, lastName: 'Lannister', firstName: 'Cersei', age: 42 },
+  { id: 3, lastName: 'Lannister', firstName: 'Jaime', age: 45 },
+  { id: 4, lastName: 'Stark', firstName: 'Arya', age: 16 },
+  { id: 5, lastName: 'Targaryen', firstName: 'Daenerys', age: null },
+  { id: 6, lastName: 'Melisandre', firstName: null, age: 150 },
+  { id: 7, lastName: 'Clifford', firstName: 'Ferrara', age: 44 },
+  { id: 8, lastName: 'Frances', firstName: 'Rossini', age: 36 },
+  { id: 9, lastName: 'Roxie', firstName: 'Harvey', age: 65 },
+];
+
+const columnGroupingModel: GridColumnGroupingModel = [
+  {
+    groupId: 'internal_data',
+    description: '',
+    children: [{ field: 'id' }],
+  },
+  {
+    groupId: 'perso',
+    children: [
+      {
+        groupId: 'naming',
+        children: [{ field: 'lastName' }, { field: 'firstName' }],
+      },
+      { field: 'age' },
+    ],
+  },
+];
+export default function BasicGroupingDemo() {
+  return (
+    <div style={{ marginTop: '20px', height: 400, width: '100%' }}>
```

Comment:
```suggestion
    <div style={{ height: 400, width: '100%' }}>
```
To keep spacing consistent across the docs